File: UrnaEletronicaAngular/src/app/Models/CandidateVotes.ts
import { Candidate } from "./Candidate";

export class CandidateVotes {

    constructor(candidate : Candidate, votesQuantity:number) {
        this.candidate = candidate;
        this.votesQuantity = votesQuantity
    }

    candidate?: Candidate;
    votesQuantity : number;
}

File: UrnaEletronicaWebAPI/Models/CandidateVotes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace UrnaEletronicaWebAPI.Models
{
    public class CandidateVotes
    {
        public CandidateVotes(Candidate? candidate, int votesQuantity)
        {
            Candidate = candidate;
            VotesQuantity = votesQuantity;
        }
        
        public Candidate Candidate { get; set; }
        public int VotesQuantity { get; set; }
        
    }
}